student/src/main/java/tds/student/web/handlers/TestResponseHandler.java

```diff
@@ -141,8 +166,17 @@ private void updateResponses (HttpServletRequest request, HttpServletResponse re
       // If scoring is done sychronously, then score and then updateDB
       // If scoring is done asynchronously, then updateDB (with score -1 and
       // status WaitingForMachineScore) and then score
+      _eventLogger.info(((EventInfo)request.getAttribute(LoggingInterceptor.EVENT_INFO))
+        .withMessage("_itemScoringService.updateResponses")
+        .withCheckpoint(SERVICE_CALL.name()));
+
       List<ItemResponseUpdateStatus> responseResults = null;
         responseResults = _itemScoringService.updateResponses (testOpp.getOppInstance (), responseReader.getResponses (), responseReader.getPageDuration());
+
+      _eventLogger.info(((EventInfo)request.getAttribute(LoggingInterceptor.EVENT_INFO))
+        .withMessage("_itemScoringService.updateResponses")
+        .withCheckpoint(SERVICE_RETURN.name()));
```

Comment:
We have a separate task to log in ItemScoringService - I'll remove this gratuitous entry for now.